### PR TITLE
Issue 28

### PR DESCRIPTION
**Pull request by JaySunSyn**
Only a demo fix. Issue 28 has been closed already.

### CODE REVIEW

**Comment by JaySunSyn** [comment]
Can be merged now.

File: demo/full-demo.html [hunk 1]
```diff
@@ -100,12 +100,14 @@
 					</cosmoz-omnitable-column>
 					<cosmoz-omnitable-column title="Custom template"
 						value-path="name" sort-on="name" width="50px" flex="2"
-						data-template="[[ customNameTemplate ]]">
+						data-template$="[[ customNameTemplate ]]">
```
Comment (nomego) [inline demo/full-demo.html]: Oh this highlights a potential pitfall.. @plequang 's naming of the property versus the element dataSet.. this should not be converted to an attribute anyway..

File: demo/full-demo.html [hunk 2]
```diff
@@ -100,12 +100,14 @@
...
 					</cosmoz-omnitable-column>
 
-					<template is="dom-repeat" items="[[ _getRowItems(selectedItems.length) ]]">
-						<paper-button>[[ item ]]</paper-button>
-					</template>
+					<div slot="actions">
+						<template  is="dom-repeat" items="[[ _getRowItems(selectedItems.length) ]]">
+							<paper-button>[[ item ]]</paper-button>
+						</template>
+					</div>
```
Comment (plequang) [inline demo/full-demo.html]: Wrapping several buttons in a single slotted element will not work with `cosmoz-bottom-bar`, which will consider the whole div as a single element in the bottom bar.
For example, bottom-bar displays normally only up to 3 buttons/elements in the toolbar, then the rest in a dropdown menu. When wrapping several buttons in `<div slot="actions">`, the bottom bar will display more then 3 buttons (try it by selecting all rows in the demo).
You should set `slot="actions"` for each buttons.

Comment (JaySunSyn) [reply demo/full-demo.html]: Oh, great, I see!

File: demo/full-demo.html (hunk 1 above)
Comment (JaySunSyn) [reply demo/full-demo.html]: omg, you are right, not sure why I did that...! Should I change the naming in the behavior or ignore the warning for now?

Comment (JaySunSyn) [reply demo/full-demo.html]: I see, there is already a new issue for renaming.

ref https://github.com/Neovici/cosmoz-omnitable/issues/32

**Comment by plequang** [comment]
Demo looks good.
However, let's hope no-one will use something similar on large data sets.
(In demo, click on the "select all" checkbox, and watch the bottom-bar animation :-) )

**Pull request merged**